util/network.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util // import "istio.io/fortio/util"
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+)
+
+const (
+	// MinTCPPort is the minimum TCP port number that can be used by Fortio http server.
+	MinTCPPort int = 1024
```

Comment:
that's not correct, I run it on port 80 at times:
https://github.com/istio/istio/blob/master/tools/setup_perf_cluster.sh#L96-L100

Reply:
OK. I wanted to ensure a user provides a valid TCP port number and I went a bit too far requiring the number be from the unreserved range. I'll update to allow all TCP port numbers.

Reply:
thanks ! lets let go listener do the validation (so allow any string)